src/ui/public/i18n/index.js

```diff
@@ -28,7 +27,4 @@ import {
 uiModules.get('i18n')
   .provider('i18n', I18nProvider)
   .filter('i18n', i18nFilter)
-  .directive('i18nId', i18nDirective)
-  .config((i18nProvider) => {
-    i18nProvider.init(metadata.translations);
```

Comment:
I removed initialization of `i18n` from here, because it will be already initialized at very beginning of loading application (https://github.com/elastic/kibana/pull/23902/files#diff-ac3204e754680bbed67c8193da8cc11bR103)